src/lib/base/component-manager.js

```diff
@@ -84,6 +91,10 @@ function decorateEvents(comp) {
         events = originalPrototype.events;
     }
 
+    if (!('ontouchstart' in window) && !events['click'] && events['touchend'] || events['tap']) {
```

Comment:
Looks like you are missing parantheses around `events['touchend'] || events['tap']` here.